requirements/pip.txt

```diff
@@ -168,17 +168,17 @@ jsonfield==3.1.0
     # via
     #   -r requirements/pip.in
     #   dj-stripe
-kombu==5.2.4
+kombu==5.3.0
```

Comment:
Nothing here, but we should double check that our Celery queue keeps working fine.